assignment4/test/lib/views/detail_screen/detail_screen.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:contacts_service/contacts_service.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:test/controller/getx_controller.dart';
+
+class ContactDetails extends StatefulWidget {
+  int index;
+  ContactDetails({super.key, required this.index});
+
+  @override
+  State<ContactDetails> createState() => _ContactDetailsState(index: index);
+}
+
+class _ContactDetailsState extends State<ContactDetails> {
+  int index;
+  _ContactDetailsState({required this.index});
+  final controller = Get.find<Mycontroller>();
+
+  TextEditingController nameController = TextEditingController(),
+      lastNameController = TextEditingController(),
+      contactController = TextEditingController();
+  @override
+  void initState() {
+    super.initState();
+
+    nameController =
```

Comment:
Find a better way to do this initialisation and use Stateless widget here

Reply:
changed it to stateless widget and modified the initialization logic